src/main/histFileMod.F90

```diff
@@ -3058,10 +3059,8 @@ subroutine hfields_1dinfo(t, mode)
           call ncd_defvar(varname='land1d_jxy', xtype=ncd_int, dim1name=namel, &
                long_name='2d latitude index of corresponding landunit', ncid=ncid)
 
-          ! --- EBK Do NOT write out indices that are incorrect 4/1/2011 --- Bug 1310
-          !call ncd_defvar(varname='land1d_gi', xtype=ncd_int, dim1name='landunit', &
-          !     long_name='1d grid index of corresponding landunit', ncid=ncid)
-          ! ----------------------------------------------------------------
+          call ncd_defvar(varname='land1d_gi', xtype=ncd_int, dim1name='landunit', &
```

Comment:
This is missing the bit of code needed from the bottom of #523. There's a do loop that loops over called to GetGlobalIndex that needs to be called. The GetGlobalIndex was added as something that can be used, but the calls to it seem to be missing.

Reply:
Yes, I marked the wrong place. @swensosc and I talked about this a bit as well. He had it correct in at least one version he had, but it was missing from the final branch. The code he showed me had it correct so he's going to add it to the right place.

Reply:
This looks correct now.